frontend/src/routes/Infrastructure/Clusters/ClusterSets/ClusterSetDetails/ClusterSetDetails.test.tsx

```diff
@@ -1018,6 +1026,7 @@ describe('ClusterSetDetails page', () => {
         await waitForTestId('credential-secret')
         await waitForNotTestId('awsAccessKeyID')
         await waitForNotTestId('awsSecretAccessKeyID')
+        await clickByLabel('Air-gapped Cluster')
```

Comment:
Probably a question for the console team, but is this the right official term we should be using? At least the docs refer to "disconnected installation", but I am not sure what the guidance for this is.

Reply:
Based on this [thread](https://coreos.slack.com/archives/CUHCCV19P/p1669822206262309) it looks like disconnected is a better term. cc @oafischer

Reply:
Also, only the first word in a heading is capitalized.

Reply:
Thanks @nyechiel @KevinFCormier, updated the label to "Disconnected cluster"